### DISCUSSION TITLE
Remove paragraph from list items (v2)

### DISCUSSION THREAD

**Discussion by Nick-Mazuk** [ueberdosis/tiptap#1341] [General Questions] [answered]
By default, all list items seem to have a paragraph inside them. I'm using the the official extensions: `@tiptap/extension-bullet-list` and `@tiptap/extension-list-item`

```html
<ul>
  <li><p>This is a list item</p></li>
</ul>
```

Is there a way to remove them? Specifically, to only allow inline elements inside of a list item?

I've tried changing the `content` attribute of the underlying schema (`content: 'inline*'`), but then I was not able to create new lists at all.

Any ideas?

**Comment by cadars** [comment, marked as answer]
I think this default behavior comes from Prosemirror, maybe this thread can help?
https://discuss.prosemirror.net/t/lists-paragraph-inside-li-instead-of-new-list-item/455

**Reply by Nick-Mazuk** [reply]
I see. Well, guess I'm going to just live with it because I don't really want to rebuild the list functionality.
